Creating a pull request after discussing with Gabriel to add official support for Google Pixel 2 (Walleye)

SudhakarJha opened the pull request:
Added walleye.json and changelog for walleye. 

Please note: I have given the download link of ROM  on android file host. As I am not sure how to add the file to Sourceforge.

SudhakarJha committed:
OTA: adding support for Google Pixel 2 (walleye)

SudhakarJha committed:
walleye: First official release changelog

gwolf2u commented on walleye.json:
```diff
@@ -0,0 +1,28 @@
+        "oem": "Google",
+        "device": "Pixel 2",
+        "filename": "crDroidAndroid-12.0-20220211-walleye-v8.2.zip",
+        "download": "https://www.androidfilehost.com/?fid=17825722713688284565",
```
use generated d/l url

SudhakarJha replied:
Done. Thank you.

SudhakarJha committed:
changed the download URL to auto generated one.

As suggested by the reviewer, changing the download URL to original (As generated by the createjason.sh)
